authentication/views.py

```diff
@@ -115,6 +115,7 @@ def post(request: Request) -> Response:
         # Check if the old password matches the user's current password
         user = request.user
         if not check_password(old_password, user.password):
+            logger.info("Invalid old password")
```

Comment:
could you please log the user too?

Reply:
otherwise it's impossible to debug that behavior and the log is almost useless

Reply:
logger.info(f"`{uname}` has inputted invalid old password")
will this work
or something like
logger.info(f"ChangePasswordView:`{uname}` inputted invalid old password")

Reply:
first one is enough! ty!